linalg.h

```diff
@@ -215,8 +215,8 @@ namespace linalg
         // NOTE: vec<T,1> does NOT have a constructor from pointer, this can conflict with initializing its single element from zero
         template<class U>
         constexpr explicit          vec(const vec<U,1> & v)             : vec(static_cast<T>(v.x)) {}
-        constexpr const T &         operator[] (int i) const            { return x; }
-        LINALG_CONSTEXPR14 T &      operator[] (int i)                  { return x; }
+        constexpr const T &         operator[] (int i) const            { (void)i; return x; }
```

Comment:
Removing `i` would have the same effect I think:
```cpp
constexpr const T & operator[] (int) const { return x; }
```
see  https://godbolt.org/z/6E5r88sdx